OloEngine/src/OloEngine/Audio/AudioListener.cpp

```
@@ -7,34 +7,34 @@
+        auto* engine = (ma_engine*)AudioEngine::GetEngine();
+        ::ma_engine_listener_set_cone(engine, m_ListenerIndex, config.ConeInnerAngle, config.ConeOuterAngle, config.ConeOuterGain);
+    }
 
-		static bool setupWorldUp = false;
-		if (!setupWorldUp)
-		{
-			::ma_engine_listener_set_world_up(engine, m_ListenerIndex, 0, 1, 0);
-			setupWorldUp = true;
-		}
-	}
+    void AudioListener::SetPosition(const glm::vec3& position) const
```

Clang (14.0.6)

[warning] method 'SetPosition' can be made static

(readability-convert-member-functions-to-static)